omod/src/main/webapp/pages/attachments.gsp

```diff
@@ -53,14 +53,20 @@ ${ ui.includeFragment("coreapps", "patientHeader", [ patient: patient ]) }
 
 <div id="att-page-main">
 
+ <% if(visit == null){ %>
+
+    <h2>Upload widget is not found because no Visit is started.</h2>
```

Comment:
Could you
1. Use a message to support i18n.
2. Share a screenshot of the end result.
?

Reply:
Could you point me to a documentation where I can find this? @mks-d  @dkayiwa @ibacher

Reply:
@reagan-meant and example [here](https://github.com/openmrs/openmrs-module-coreapps/blob/5789accfe48bfd7071b1abad6021ead1da325131/omod/src/main/webapp/fragments/patientdashboard/visitIncludes.gsp#L41) in Core Apps.

Reply:
Thanks @mks-d....The following commit was before i realised Ui framework does that in one line....Was trying to do it manually.Lemme send a cleaner commit now

Reply:
Screenshot is added to the jira ticket